xrmocap/model/registrant/smplifyx.py

```diff
@@ -53,25 +71,81 @@ def get_keypoint_weight(self,
                 Weight of body keypoints. Body part segmentation
                 definition is included in the HumanData convention.
                 Defaults to 1.0.
+            hand_weight (float, optional):
+                Weight of hands keypoints. Body part segmentation
+                definition is included in the HumanData convention.
+                Defaults to 1.0.
+            face_weight (float, optional):
+                Weight of face keypoints. Body part segmentation
+                definition is included in the HumanData convention.
+                Defaults to 1.0.
+            shoulder_weight (float, optional):
+                Weight of shoulder keypoints. Body part segmentation
+                definition is included in the HumanData convention.
+                Defaults to None.
+            hip_weight (float, optional):
+                Weight of hip keypoints. Body part segmentation
+                definition is included in the HumanData convention.
+                Defaults to None.
+            foot_weight (float, optional):
+                Weight of feet keypoints. Body part segmentation
+                definition is included in the HumanData convention.
+                Defaults to None.
 
         Returns:
-            torch.Tensor: Per keypoint weight tensor of shape (K).
+            torch.Tensor: _description_
```

Comment:
_description_

Reply:
Fixed in [b867ad1](https://github.com/openxrlab/xrmocap/pull/52/commits/b867ad1f23a1edab97d0111ff6943578a569535b).